examples/web-wagmi/src/publications/UseCreateMirror.tsx

```diff
@@ -33,14 +33,13 @@ function MirrorInner({ publisher }: MirrorInnerProps) {
     return <p>Can't mirror a mirror</p>;
   }
 
-  const isMirroredByMe = publication.isOptimisticMirroredByMe || publication.mirrors.length > 0;
-
   return (
     <div>
       <PublicationCard publication={publication} />
 
       <div>Total Mirrors: {publication.stats.totalAmountOfMirrors}</div>
-      <div>Is Mirrored by Me: {isMirroredByMe ? 'true' : 'false'}</div>
+      <div>My mirrors: {publication.myMirrors.length}</div>
```

Comment:
❤️

Reply:
I'd just leave them as `mirrors` then, less maintenance, and just explain that they are observer mirrors in the docs, unfortunately, we can't really auto-document fragment fields

Reply:
ok

Reply:
removed `myMirrors` for now